cli/cli.go

```diff
@@ -190,36 +221,49 @@ func Priam(args []string, defaultCfgFile string, infoW, errorW io.Writer) {
 		cli.StringFlag{Name: "given", Usage: "given name of the user account"},
 	}
 
+	templateFlags := []cli.Flag{
+		cli.IntFlag{Name: "accessTokenTTL", Usage: "seconds that the access token is valid", Value: 480},
+		cli.StringFlag{Name: "authGrantTypes", Value: "authorization_code"},
+		cli.BoolFlag{Name: "displayUserGrant", Usage: "prompt for user consent when registering app instance"},
+		cli.IntFlag{Name: "length", Value: 32},
+		cli.StringFlag{Name: "redirectUri", Value: "horizonapi://oauth2"},
+		cli.IntFlag{Name: "refreshTokenTTL", Usage: "seconds that the refresh token is valid", Value: 2628000},
+		cli.StringFlag{Name: "resourceUuid", Value: "00000000-0000-0000-0000-000000000000"},
+		cli.StringFlag{Name: "scope", Value: "user profile email"},
+		cli.StringFlag{Name: "tokenType", Value: "Bearer"}}
+
+	clientFlags := []cli.Flag{
+		cli.IntFlag{Name: "accessTokenTTL", Usage: "seconds that the access token is valid", Value: 480},
+		cli.StringFlag{Name: "authGrantTypes", Value: "authorization_code"},
+		cli.BoolFlag{Name: "displayUserGrant", Usage: "prompt for user consent"},
+		cli.BoolFlag{Name: "inheritanceAllowed"},
+		cli.BoolFlag{Name: "internalSystemClient"},
+		cli.StringFlag{Name: "redirectUri", Value: "horizonapi://oauth2"},
+		cli.IntFlag{Name: "refreshTokenTTL", Usage: "seconds that the refresh token is valid", Value: 2628000},
+		cli.StringFlag{Name: "rememberAs"},
+		cli.StringFlag{Name: "resourceUuid", Value: "00000000-0000-0000-0000-000000000000"},
+		cli.StringFlag{Name: "scope", Value: "user profile email"},
+		cli.StringFlag{Name: "secret"},
+		cli.StringFlag{Name: "strData"},
+		cli.IntFlag{Name: "tokenLength", Value: 32},
+		cli.StringFlag{Name: "tokenType", Value: "Bearer"},
+	}
+
 	app.Commands = []cli.Command{
 		{
 			Name: "app", Usage: "application publishing commands",
 			Subcommands: []cli.Command{
 				{
-					Name: "add", Usage: "add applications to the catalog", ArgsUsage: "[./manifest.yaml]",
-					Action: func(c *cli.Context) error {
-						if args, ctx := initCmd(cfg, c, 0, 1, true, nil); ctx != nil {
```

Comment:
Here the arg was optional, and now -if I read that correctly- it's not anymore if you call cmdWithAuth1Arg(...), am I right?